apps/game_backend/lib/game_backend/users/currencies.ex

```diff
@@ -48,8 +48,8 @@ defmodule GameBackend.Users.Currencies do
 
   @doc """
   Gets a single currency.
-
-  Returns nil if the Currency does not exist.
+  Returns {:ok, currency} if query succeeds.
+  Returns {:error, :not_found} if the Currency does not exist.
```

Comment:
I think we are repeating the return of this function here and in the examples, we should delete one of them, also this change makes the example of a non existing currency wrong

Reply:
I'm not sure I get what you meant. I changed the example because it was outdated. Let me know if there's something else.